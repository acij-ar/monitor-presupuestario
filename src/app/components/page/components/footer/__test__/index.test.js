const React = require('react');
const { shallow } = require('enzyme');

const Footer = require('..');

describe('Footer component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
