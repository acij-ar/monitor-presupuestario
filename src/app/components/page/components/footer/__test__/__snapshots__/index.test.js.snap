// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<div
  id="footer"
>
  <div
    id="footer-content"
  >
    <span
      id="acij-initiative"
    >
      UNA INICIATIVA DE
    </span>
    <a
      href="http://acij.org.ar/"
      id="acij-link"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="ACIJ"
        src="/static/logo-acij-small.png"
      />
    </a>
    <a
      className="social-link"
      href="https://twitter.com/ACIJargentina"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="twitter"
        src="/static/twitter.svg"
      />
    </a>
    <a
      className="social-link"
      href="https://www.facebook.com/ACIJ.ORG"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="facebook"
        src="/static/facebook.svg"
      />
    </a>
    <a
      className="social-link"
      href="https://github.com/acij-ar/monitor-presupuestario"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="github"
        src="/static/github.svg"
      />
    </a>
  </div>
</div>
`;
